### PR TITLE
Fix for tests breaking after tee.

### PR DESCRIPTION
**Pull request by j-bennet**
## Description

This is an attempt to fix the `tee` test breaking: https://github.com/dbcli/pgcli/issues/795.

Also, this fixes the error in `notee` test which should not be `notee` (that's a mycli command) but `\o`: https://github.com/dbcli/pgcli/issues/797.

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [X] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ python:
   - "3.5"
   - "3.6"
 
-install: 
-  - pip install . docutils pytest mock codecov==1.5.1 behave pexpect==3.3
+install:
+  - pip install . docutils pytest mock codecov==1.5.1 behave pexpect==3.3 retrying>=1.3.3
   - pip install git+https://github.com/hayd/pep8radius.git
```
Comment (tsroten) [inline .travis.yml]: I think you could probably replace both of these pip install lines with these two so we don't have to keep them up-to-date. Just add `docutils` and `codecov` to the requirements file.
```
pip install .
pip install -r requirements-dev.txt
```

Comment (j-bennet) [reply .travis.yml]: Good point.

File: tests/features/environment.py [hunk 2]
```diff
@@ -134,6 +134,7 @@ def after_scenario(context, _):
                 '{0}> '.format(dbname),
                 timeout=5
             )
+        context.cli.sendcontrol('u')
```
Comment (meeuw) [inline tests/features/environment.py]: for mycli I had to use `sendcontrol('c')` to have always a "clean" prompt... ctrl-u removed the current line and there might be a multi-line statement in the prompt...

Comment (j-bennet) [reply tests/features/environment.py]: Well, after scenario is done, there shouldn't be anything in prompt, that's the point.

Comment (meeuw) [reply tests/features/environment.py]: while using ctrl-u there might be something left at the prompt though...

Comment (j-bennet) [reply tests/features/environment.py]: Didn't know that. In that case I better switch to Ctrl + C.

**Pull request merged**